testing/test_junitxml.py

```diff
@@ -693,17 +708,13 @@ def test_fail():
         """
         )
         result, dom = run_and_parse("-o", f"junit_logging={junit_logging}")
-        node = dom.find_first_by_tag("testsuite")
-        tnode = node.find_first_by_tag("testcase")
-        if junit_logging in ["system-err", "out-err", "all"]:
-            assert len(tnode.find_by_tag("system-err")) == 1
-        else:
-            assert len(tnode.find_by_tag("system-err")) == 0
+        node = dom.get_first_by_tag("testsuite")
+        tnode = node.get_first_by_tag("testcase")
 
-        if junit_logging in ["log", "system-out", "out-err", "all"]:
-            assert len(tnode.find_by_tag("system-out")) == 1
-        else:
-            assert len(tnode.find_by_tag("system-out")) == 0
+        has_logging = junit_logging in ["system-err", "out-err", "all"]
+        expected_output_len = 1 if has_logging else 0
+        print(tnode)
+        assert len(tnode.find_by_tag("system-err")) == expected_output_len
```

Comment:
Not checking `system-out` anymore now, only `system-err`?

Reply:
good catch, i missed that in the rebase